cumulusci/tasks/salesforce/org_settings.py

```diff
@@ -38,21 +39,24 @@ def _init_options(self, kwargs):
 
     def _run_task(self):
         settings = {}
+        object_settings = {}
         if self.options.get("definition_file"):
             with open(self.options["definition_file"], "r") as f:
                 scratch_org_definition = json.load(f)
                 settings = scratch_org_definition.get("settings", {})
+                object_settings = scratch_org_definition.get("objectSettings", {})
 
         dictmerge(settings, self.options.get("settings", {}))
+        dictmerge(object_settings, self.options.get("object_settings", {}))
```

Comment:
Ha! Yeah, that's what I get for mixing up `pathlib` and EAFP comments. 
```suggestion
        settings = self.options.get("settings", {})
        object_settings = self.options.get("object_settings", {})

        try:
            path = self.options["definition_file"]
            scratch_org_definition = json.loads(Path(path).read_text())
        except KeyError:
            scratch_org_definition = {}

        dictmerge(scratch_org_definition.get("settings", {}), settings)
        dictmerge(scratch_org_definition.get("objectSettings", {}), object_settings)
```

Reply:
Yeah, but this change reverses the priority of merging between the scratch org def and the user-supplied options. The existing code is inelegant, but I'd prefer to keep it largely as-is to avoid changing behavior in very-difficult-to-debug ways.

Reply:
Applied the change.

Reply:
Probably should have been clear that this wasn't commit-able as-is. 🙇🏾